### ISSUE TITLE
Spurious OSSEC alerts due to Apache AppArmor profile

### ISSUE THREAD

**Issue by garrettr** [freedomofpress/securedrop#1331]
When apache2 starts the WSGI application, it tries to exec `uname`, which triggers an AppArmor denial, which in turn triggers an OSSEC alert:

```
OSSEC HIDS Notification.
[timestamp redacted]

Received From: (app) [ip address redacted]->/var/log/syslog
Rule: 100012 fired (level 7) -> "Apparmor denied event"
Portion of the log(s):

[timestamp redacted] app kernel: [[redacted]] audit: type=1400 audit([redacted]): apparmor="DENIED" operation="exec" profile="/usr/sbin/apache2" name="/bin/uname" pid=3989 comm="sh" requested_mask="x" denied_mask="x" fsuid=33 ouid=0

--END OF NOTIFICATION
```

It might be worthwhile to investigate the cause of this alert, since I do not believe this was an issue when 0.3 was released. We did a thorough audit to find (and write rules to avoid) spurious OSSEC alerts around that time. My hunch is that apache or some component related to it has been updated since then, and the update changed the behavior in such a way as to cause the AppArmor denial that we are seeing today.

This issue was noticed during QA testing for 0.3.7, but we decided not to fix it then because we wanted to avoid additional scope creep for 0.3.7. It has recently also been reported to us by an admin as a cause of OSSEC alert spam, which indicates to us that a fix should be prioritized for the next point release.

**Comment by fowlslegs** [comment]
I'll write an OSSEC rule for this for 0.3.8. Must be due to an update in Apache.

**Comment by ageis** [comment]
@fowlslegs Isn't the proper thing to do update the AppArmor profile?

**Comment by garrettr** [comment]
@ageis that's correct, you should update the AppArmor profile so it allows apache to execute uname.

**Comment by fowlslegs** [comment]
Yes, thanks for the correction.

**Comment by conorsch** [comment]
Related: #1240.

**Comment by fowlslegs** [comment]
Resolved by #1332.
